mod/consensus/pkg/cometbft/service/service.go

```diff
@@ -54,7 +53,10 @@ const (
 	execModeFinalize
 )
 
-const InitialAppVersion uint64 = 0
+const (
+	initialAppVersion uint64 = 0
+	appName           string = "beacond"
```

Comment:
I prefer the name here since it's constant. Happy to revert the change if we don't agree with it